docs/sources/menu.yaml

```diff
@@ -165,6 +165,19 @@
       name: Queries
     - link: /panels/transformations/
       name: Transformations
+      children:
+        - link: /panels/transformations/
+          name: Overview
+        - link: /panels/transformations/prereq/
+          name: Prerequisites
+        - link: /panels/transformations/apply/
+          name: Apply a transformation
+        - link: /panels/transformations/types-options/
+          name: Transformation types and options
+        - link: /panels/transformations/debug/
+          name: Debug transformations
+        - link: /panels/transformations/remove/
+          name: Remove a transformation
```

Comment:
I agree with @torkelo that this split might be a little too much. Prerequisites, Apply a transformation, Debug, Remove a transformation could be part of the Overview page while Transformation types and options could be the only separate page. What do you think?

Reply:
We like to separate documentation "types" when possible, such as concept, reference, and task. In this case, the use case, explanation of what transformations are, and prerequisites could all be classified as "concept." Debugging and removing transformations are tasks, so we would want to group them with other tasks. But yes, I agree that the current structure is too granular, and I made some suggestions in my earlier review.

Reply:
Thank you @torkelo and @dprokop for your suggestions. I have moved the "Prerequisites" section to the Overview section. Also, I moved the debug and remove sections to the Apply Transformations topic. 

We have three topics now. Hope this is OK.

Reply:
@oddlittlebird, I have made the suggested changes. Please review and let me know. Thanks!